docs/dev/glossary/glossary.md

```diff
@@ -0,0 +1,69 @@
+# Overview
+
+This section contains a comprehensive list of terms associated to zkSync and it's related ecosystem.
```

Comment:
this page doesn't show on the left-hand panel (unless I missed it) -- is it intended?

Reply:
It's a work in progress.